cli/common.go

```diff
@@ -63,12 +63,12 @@ func commonFlags(f *pflag.FlagSet) {
 	f.StringVar(&cfg.ForceKubeVersion, "force-kube-version", "", "force a kubernetes version, rather than determining from the apiserver")
 	f.BoolVar(&cfg.NoVerifySig, "no-verify-signatures", false, "don't gpg-verify all downloaded addons")
 	f.StringVar(&cfg.GpgKeyringPath, "keyring", "/pubring.gpg", "path to the gpg keyring")
-	f.StringVar(&cfg.VersionManifestPath, "version-manifest", "/versions.yaml", "path to the version manifest file")
+	f.StringVar(&cfg.VersionManifestPath, "version-manifest", "", "path to the runtime-mappings manifest file")
 	f.StringVar(&verbose, "verbose", "info", "verbosity level")
 }
 
 // parseFlags parses CLI options, returning a populated configuration for the bootstrap agent
-func parseFlags() (internal.Config, error) {
+func parseFlags(runtimeMappingsPath string) (internal.Config, error) {
```

Comment:
can you call this something like "defaultRuntimeMappingsPath"? Just so it's clear that this is a fallback.

Reply:
Ack. I've also left a docstring note to describe argument usage.